source/users.rst

```diff
@@ -15,10 +36,49 @@ PostgreSQL Operator comes with the following users:
       - Superuser, Create role, Create DB, Replication, Bypass RLS
     * - ``primaryuser``
       - Replication
-    * - ``testuser``
-      - 
+    * - ``pguser``
+      - Non-privileged user
+    * - ``pgbouncer``
+      - Еhe administrative user for the `pgBouncer connection pooler <http://pgbouncer.github.io/>`_
 
 The ``postgres`` user will be the admin user for the database instance. The
 ``primaryuser`` is used for replication between primary and replicas. The
-``testuser`` is a normal user that has access to the database “userdb” that is
-created for testing purposes.
+``pguser`` is the default non-privileged user.
```

Comment:
Please mention that the name of the user can be configured in CR under spec.user.